NOTICE.txt

```diff
@@ -2781,6 +2781,35 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+--------------------------------------------------------------------
+Dependency: github.com/ugorji/go
```

Comment:
The delay on this PR may have a silver lining.  [__fxamacker/cbor__](https://github.com/fxamacker/cbor/blob/master/README.md) wasn't available in Jan 2019.

![alt text](https://github.com/fxamacker/images/raw/master/cbor/v2.2.0/cbor_security_table.svg?sanitize=1 "CBOR Security Comparison")

For more info about CBOR security considerations, see [__Section 8 of RFC 7049__](https://tools.ietf.org/html/rfc7049#section-8).

__Click to expand:__

<details>
 <summary>Program Size Comparison</summary><p>

__`fxamacker/cbor`__ produces smaller binaries.

![alt text](https://github.com/fxamacker/images/raw/master/cbor/v2.2.0/cbor_app_size_barchart.svg?sanitize=1 "CBOR program size comparison bar chart")

</details>

<details>
 <summary>Speed & Memory Comparison</summary><p>

[__fxamacker/cbor__](https://github.com/fxamacker/cbor/blob/master/README.md) is faster and more memory efficient when using CBOR data with Go structs:

![alt text](https://github.com/fxamacker/images/raw/master/cbor/v2.2.0/cbor_speed_table.svg?sanitize=1 "CBOR Speed Comparison")

Benchmarks used example data from [RFC 8392 Appendix A.1](https://tools.ietf.org/html/rfc8392#appendix-A.1) and default options for CBOR libraries.

Speed is only one factor and it shouldn't be the only criteria.  It's more important to produce correct results and not crash.  fxamacker/cbor passes 375+ tests and uses coverage-guided fuzzing with 1,100+ CBOR files.

</details>

<details>
 <summary>CBOR API is encoding/json + keyasint/toarray/omitempty struct tags</summary><p>

 [__fxamacker/cbor__](https://github.com/fxamacker/cbor/blob/master/README.md) provides identical API to Go's encoding packages, plus struct tags like `toarray` and `keyasint`.

__Standard API__: functions with signatures identical to [`encoding/json`](https://golang.org/pkg/encoding/json/) include:  
`Marshal`, `Unmarshal`, `NewEncoder`, `NewDecoder`, `encoder.Encode`, and `decoder.Decode`.

__Standard interfaces__ allow custom encoding or decoding:  
`BinaryMarshaler`, `BinaryUnmarshaler`, `Marshaler`, and `Unmarshaler`.

__Struct tags__ like __`toarray`__ & __`keyasint`__ translate Go struct fields to CBOR array elements, etc.

<br>

[![CBOR API](https://github.com/fxamacker/images/raw/master/cbor/v2.2.0/cbor_struct_tags_api.svg?sanitize=1)](#usage)